.github/workflows/build-images.yaml

```diff
@@ -122,7 +122,7 @@ jobs:
           image: kuadrant-operator-catalog
           tags: ${{ env.IMG_TAGS }}
           dockerfiles: |
-            ./index.Dockerfile
+            ./catalog.Dockerfile
```

Comment:
Then, what is the use of dockerfile generation tasks `make catalog-generate` ?

Reply:
what about `make catalog-build` ?

Reply:
`make catalog-build` could be removed, but I'd rather leave it there since it will work on OpenShift I reckon